fluentgen/src/main/java/com/azure/autorest/fluent/util/FluentUtils.java

```diff
@@ -345,6 +353,18 @@ public static IType getValueTypeFromResponseType(IType clientType) {
         return bodyType;
     }
 
+    private static IType getValueTypeFromResponseTypeSubType(GenericType type) {
+        IType bodyType = null;
+        if (ResponseBase.class.getSimpleName().equals(type.getName())) {
+            bodyType = type.getTypeArguments()[1];
+        } else if (SimpleResponse.class.getSimpleName().equals(type.getName())) {
+            bodyType = type.getTypeArguments()[0];
+        } else if (StreamResponse.class.getSimpleName().equals(type.getName())) {
+            bodyType = GenericType.FLUX_BYTE_BUFFER;
+        }
+        return bodyType;
```

Comment:
Wondering whether fallback to `type.getTypeArguments()[1]`?

Reply:
Sure. I'm not sure if there's case where there's no type arguments(like StreamResponse). Better to throw exception when such case happens anyway.